src/java/org/apache/cassandra/tracing/Tracing.java

```diff
@@ -329,7 +357,7 @@ public Map<ParamType, Object> addTraceHeaders(Map<ParamType, Object> addToMutabl
 
         addToMutable.put(ParamType.TRACE_SESSION, Tracing.instance.getSessionId());
         addToMutable.put(ParamType.TRACE_TYPE, Tracing.instance.getTraceType());
-        String keyspace = Tracing.instance.getKeyspace();
+        String keyspace = Tracing.instance.get().tracedKeyspace();
```

Comment:
Is `Tracing.instance.getKeysapce()` still needed with this change? I only see one other usage in `TracingTest` and maybe that should be updated to match this change?

Reply:
Good call, @djatnieks. I'll delete `Tracing.getKeyspace()` and use the new `tracedKeyspace()` method instead.